pom.xml

```diff
@@ -112,6 +114,11 @@
                 <artifactId>owl-extension</artifactId>
                 <version>${tinkar-core.version}</version>
             </dependency>
+            <dependency>
+                <groupId>dev.ikm.tinkar</groupId>
+                <artifactId>tinkar-data-maven-plugin</artifactId>
+                <version>1.4.0-SNAPSHOT</version>
```

Comment:
Rather than using a snapshot, the version should be ${tinkar-starter-data.version}